packages/core/src/main/Account.ts

```diff
@@ -85,6 +93,7 @@ class Account extends EventEmitter {
     notification: NotificationService;
     self: SelfService;
   };
+  private readonly assetOriginalCache: {[messageId: string]: Original} = {};
```

Comment:
`assetOriginalCache` can be removed. Instead we can emit `Account.INCOMING.IMAGE` & `Account.INCOMING.IMAGE_META`.